src/RouteConfig.js

```diff
@@ -69,6 +69,8 @@ export default (
     model: new OAuthModel(container.constitute('UserRepository')),
   });
 
+  container.bindValue('OauthServer', oauth);
```

Comment:
The oauth server should always be the same. No one will ever want to override any of the implementation. Why don't you store the object in constitute?